src/QMCWaveFunctions/EinsplineSetBuilderCommon.cpp

```diff
@@ -769,8 +769,10 @@ EinsplineSetBuilder::OccupyBands(int spin, int sortBands, int numOrbs)
   if (myComm->rank() != 0) return;
   if(spin>=NumSpins)
   {
-    app_error() << "Invalid request for orbitals in spin group " << spin
-                << ". Current h5 file only contains spin groups " << "[0.." << NumSpins-1 << "]." << std::endl;
+    //app_error() << "Invalid request for orbitals in spin group " << spin
```

Comment:
Please remove commented-out code. 

Put in a real comment or expand the error message to address user and developer.